lib/usd/ui/USDImportDialogCmd.cpp

```diff
@@ -66,6 +76,69 @@ void* USDImportDialogCmd::creator()
 	return new USDImportDialogCmd();
 }
 
+MStatus USDImportDialogCmd::applyToProxy(const MString& proxyPath)
+{
+	MDagPath proxyShapeDagPath;
+	MSelectionList selection;
+	selection.add(proxyPath);
+	MStatus status = selection.getDagPath(0, proxyShapeDagPath);
+	if (status.error())
+		return status;
+
+	MObject proxyShapeObj = proxyShapeDagPath.node(&status);
+	CHECK_MSTATUS_AND_RETURN(status, status);
+	if (status.error())
+		return status;
```

Comment:
Here you can use CHECK_MSTATUS_AND_RETURN_IT(status) and you don't need the extra check for .error() since the macro returns if status != kSuccess.